data/json/items/gun/50.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "id": "m107a1",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Barrett M107A1",
+    "name_plural": "Barrett M107A1",
+    "description": "A large, shoulder-fired, .50 caliber anti-materiel rifle.  Its large size, recoil, and noise is offset by its damage and range." ,
+    "price": 650000, "//" : "Minimum list price for the bolt-action, Gun Digest, 2012.",
+    "material": "steel",
+    "ammo": "50",
+    "weight": 12602,
+    "volume": 17,
+    "bashing": 12,
+    "to_hit": -1,
+    "ranged_damage": -5,
+    "range": 45,
+    "sight_dispersion": 60,
+    "recoil": 50,
+    "durability": 8,
+    "barrel_length" : 5,
+    "magazine_well": 1,
+    "magazines": [ [ "50", [ "m107a1mag" ] ] ],
+    "flags": "NEVER_JAMS"
+  },
+  {
+    "id": "m2browning",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "M2 Browning HMG",
+    "name_plural": "M2 Browning HMG",
+    "description": "A heavy machine gun used by the US Military from its inception to the cataclysm, and even rarely by cataclysm survivors.  Its massive size and design make it impossible to use unless deployed or mounted to a vehicle.",
+    "price": 1800000,
+    "material": "hardsteel",
+    "skill": "rifle",
+    "ammo": "50",
+    "weight": 38000,
+    "volume": 20,
+    "bashing": 12,
+    "to_hit": -1,
+    "range": 45,
+    "dispersion": 30,
+    "sight_dispersion": 75,
+    "aim_speed" : 5,
+    "recoil": 225,
+    "durability": 8,
+    "reload": 400,
+    "burst": 25,
```

Comment:
Wikipedia says that "M2HB can be selected to fire single-shots or at less than 40 rounds per minute, or rapid fire for more than 40 rounds per minute". So it's supposed to have single-fire mode unless it is not a HB variant.